chirp-impl/src/test/java/sample/chirper/chirp/impl/ChirpServiceTest.java

```diff
@@ -33,8 +33,9 @@
     // Creating a new persistent entity is slow in some runtime environment (e.g. free tier Travis) causing the
     // tests to fail on timeout. First time we invoke a call that requires a persistent entity we will
     // wait a bit longer.
-    private static int firstRequestOnEntityTimeout;
-    private static final int secondRequestOnEntityTimeout = 1;
+    private static int persistentEntityDefault;
+    private static final int probeRequestTimeout = 10;
+    private static final int serviceInvocationTimeout = 3;
```

Comment:
I renamed the timeouts as suggested by @renatocaval  and went a bit farther:

* extract some more magic numbers from the code and place them all colocated.
* extracted `addChirp` method to simplify code and reduce duplication